packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -909,11 +917,15 @@ class RectangularSliderTrackShape extends SliderTrackShape {
     }
 
     // Used to create a gap around the thumb iff the slider is disabled.
+    // If the slider is enabled, the track can be drawn beneath the thumb
+    // without gaps. But when the slider is disabled, the track is shortened
+    // and this gap helps determine how much shorter it should be.
+    // TODO(clocksmith): The new material spec has a gray circle in place of this gap.
     double horizontalAdjustment = 0.0;
     if (!isEnabled) {
-      final double thumbRadius = sliderTheme.thumbShape.getPreferredSize(isEnabled, isDiscrete).width / 2.0;
-      final double gap = _thumbGap * (1.0 - enableAnimation.value);
-      horizontalAdjustment = thumbRadius + gap;
+      final double disabledThumbRadius = sliderTheme.thumbShape.getPreferredSize(false, isDiscrete).width / 2.0;
```

Comment:
Nice

Reply:
👍 This may get even easier once this is updated to spec